src/js/providers/html5.js

```diff
@@ -837,7 +838,25 @@ define([
         this.getName = function() {
             return { name : _name };
         };
+        this.setCurrentAudioTrack = _setCurrentAudioTrack;
 
+        function _setAudioTracks(tracks) {
+            if(tracks && tracks.length > 0) {
+                _currentAudioTrackIndex = 0;
+                tracks[_currentAudioTrackIndex].enabled = true;
```

Comment:
Is it necessary to enable the first track? The media might define any track as the default track, and we should not change that when getting the list of available tracks.